architecture/infrastructure_components/web_console.adoc

```diff
@@ -137,6 +137,17 @@ link:../core_concepts/deployments.html#replication-controllers[replication
 controller].
 ====
 
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+http://cockpit-project.org[Cockpit] is automatically installed and enabled in
+{product-title} 3.1 and beyond to help you monitor your development environment.
```

Comment:
A quick grep shows three hits for "[0-9] and later" and only one for "[0-9] and beyond", so maybe "and later" is (slightly) preferred.  Personally, i prefer "and later" since "and beyond" brings to mind [old memories](https://en.wikipedia.org/wiki/Toy_Story#.22To_Infinity_and_Beyond.22).